casper/src/main/rholang/WalletCheck.rho

```diff
@@ -31,9 +31,9 @@ contract @["WalletCheck", "create"](@hash, @purse) = {
                 new hashOut in {
 
                   // Check that given pubKey is the preimage of hash
-                  @"keccak256Hash"!(pubKey.toByteArray(), *hashOut) |
+                  algorithm!(pubKey.toByteArray(), *hashOut) |
```

Comment:
The `algorithm` field in the `Wallet` case class was originally the signature algorithm. Interpreting it as the hashing algorithm is probably fine, but if the point of these "pre-wallets" is to always be based on Ethereum addresses, then maybe we should just remove that degree of freedom.

Reply:
Done.